conf/routes

```diff
@@ -81,11 +81,15 @@ GET  /$country<(uk|us|au|int)>/subscribe/premium-tier   controllers.Subscription
 GET  /subscribe/weekly                            controllers.Subscriptions.weeklyGeoRedirect()
 GET  /$country<(uk|us|au|int|nz|ca|eu)>/subscribe/weekly   controllers.Subscriptions.weekly(country: String)
 
-GET  /subscribe/paper          controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = false)
-GET  /subscribe/paper/delivery          controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = true)
-GET  /uk/subscribe/paper          controllers.Subscriptions.paper(withDelivery: Boolean = false)
-GET  /uk/subscribe/paper/delivery          controllers.Subscriptions.paper(withDelivery: Boolean = true)
-
+GET  /subscribe/paper                             controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = false)
+GET  /subscribe/paper/delivery                    controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = true)
+GET  /uk/subscribe/paper                          controllers.Subscriptions.paper(withDelivery: Boolean = false)
+GET  /uk/subscribe/paper/delivery                 controllers.Subscriptions.paper(withDelivery: Boolean = true)
+
+GET  /subscribe/print                             controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = false)
+GET  /subscribe/print/delivery                    controllers.Subscriptions.paperMethodRedirect(withDelivery: Boolean = true)
+GET  /uk/subscribe/print                          controllers.Subscriptions.paper(withDelivery: Boolean = false)
```

Comment:
we should make one of these routes be redirects so as to not have dupe pages SEO-wise

Reply:
Yeah, I opted against that as it causes another handshake for the client. Though I suspect the `print` endpoints will be used exclusively internally (if not just by me) so it probably doesn't matter.